### ISSUE TITLE
Notification object should not return an array

### ISSUE THREAD

**Issue by janjongboom** [ARMmbed/mbed-connector-api-node#12]
Notification object currently looks like this:

```js
[ { ep: 'bfc164a6-afea-4d72-9e34-73f6b854189c',
    path: '/accelerometer/0/last_knock',
    payload: '7622',
    'max-age': 60 } ]
```

It should not return an array, but rather emit multiple notifications.

**Comment by bridadan** [comment]
I actually went back and forth on this myself whether to do an array or multiple notifications.

The way I rationalized it at the time was perhaps it would be useful to the user to get multiple at a time so they could have the option to do batch operations?

Not sure if this makes sense though, I'm very open to counterarguments.

**Comment by janjongboom** [comment]
You still get the notifications in one tick so not wasting CPU time, but sending an array does not make sense in node if you have an event emitter in place anyway.

**Comment by bridadan** [comment]
Make sense to me, I'll make the change

**Comment by bridadan** [comment]
Fixed in b4809c9961692105c080d2c1379ec0af49989e9d, added to release 2.0.0
